marathon_acme/tests/fake_marathon.py

```diff
@@ -94,6 +112,24 @@ def get_event_subscriptions(self, request):
         request.setResponseCode(200)
         return self._json_response(request, response)
 
+    @app.route('/v2/eventSubscriptions', methods=['POST'])
+    def post_event_subscriptions(self, request):
+        query = urisplit(request.uri).getquerydict()
+
+        assert 'callbackUrl' in query
+        assert query['callbackUrl']
+
+        callback_url = query['callbackUrl'][0]
+        self._marathon.add_event_subscription(callback_url)
+
+        # FIXME: Should this be triggered by add_event_subscription()?
```

Comment:
My only concern was having to pass through the client IP seemed like responsibilities were getting muddled but I guess it's not that bad.